### DISCUSSION TITLE
Did XQuartz 2.8.2 pick up the X.org version 21.1.4 updates

### DISCUSSION THREAD

**Discussion by nbriggs** [XQuartz/XQuartz#271] [Q&A] [answered]
It's hard to tell from the XQuartz "About" menu item whether it picked up any of the latest X.org 21.1.4 updates -- which seem to have arrived around July 13th, about the same time as XQuartz 2.8.2...

See https://www.theregister.com/2022/07/13/xorg_servers_updated/

**Comment by jeremyhu** [comment, marked as answer]
No.  Neither does 2.8.3_beta1.  The next release (probably 2.8.3_rc1) should pick it up.
